test/unit/test_rtp.py

```diff
@@ -0,0 +1,247 @@
+import pydarn
+import unittest
+import matplotlib.pyplot as plt
+from datetime import datetime
+
+
+class TestRTP(unittest.TestCase):
+    """
+    Testing class RTP, due to the plotting nature all plots will be saved
+    since there is return testing
+    """
+    @classmethod
+    def setUpClass(cls):
+        """
+        Runs once before all tests are ran. Loads and reads in the
+        fitacf file that will be used in all the unit tests.
+        """
+        #fitacf_file = "../testfiles/20190203.0001.00.fhw.fitacf3"
+        fitacf_file = "../testfiles/20180220.C0.rkn.fitacf"
+        darn_read = pydarn.DarnRead(fitacf_file)
+        cls.fitacf_data = darn_read.read_fitacf()
+
+    def test_simple_range_time_plot(self):
+        """
+        plots a simple elevation range-time plot for beam 7
+        """
+        pydarn.RTP.plot_range_time(self.fitacf_data, parameter='elv',
+                                   beam_num=7,
+                                   boundary=(0, 57))
+        plt.title("Simple Elevation no ground scatter, beam 7 Saskatoon plot")
+        plt.show()
+
+    def test_ground_scatter_range_time_plot(self):
+        """
+        plots a simple elevation rang-time plot with ground scatter
+        """
+        pydarn.RTP.plot_range_time(self.fitacf_data, parameter='elv',
+                                   beam_num=7, ground_scatter=True)
+        plt.title("Elevation with Ground Scatter with beam 7")
+        plt.show()
+
+    def test_velocity_range_time_plot(self):
+        """
+        plots a velocity range-time plot for
+        beam 5 with the reversed jet colour mapping
+        """
+        pydarn.RTP.plot_range_time(self.fitacf_data, parameter='v',
+                                   beam_num=5, color_map='jet_r')
+        plt.title("Velocity with reversed jet color map")
+        plt.show()
+
+    def test_velocity_error_range_time_plot(self):
+        """
+        plots velocity error range time plot which is not a pre-set
+        parameter also added extra settings for the color bar.
+        """
+        pydarn.RTP.plot_range_time(self.fitacf_data, parameter='v_e',
+                                   beam_num=5, color_map='jet_r',
+                                   color_bar_label="velocity error $m/s$",
+                                   boundary=(-200, 200))
+        plt.title("Non-standard parameter: Velocity error\n"
+                  " with reversed jet color map")
+        plt.show()
+
+    def test_power_range_time_plot(self):
+        """
+        plots power range-time with ground scatter
+        """
+        pydarn.RTP.plot_range_time(self.fitacf_data, parameter='p_l',
+                                   beam_num=7, ground_scatter=True)
+        plt.title("Elevation with Ground Scatter")
```

Comment:
Change Ground Scatter to either Ground Backscatter or Groundscatter

Reply:
Done